kafka/protocol/api.py

```diff
@@ -65,3 +68,30 @@ def API_VERSION(self):
     def SCHEMA(self):
         """An instance of Schema() representing the response structure"""
         pass
+
+    def to_object(self):
+        return _to_object(self.SCHEMA, self.fields)
+
+
+def _to_object(schema, data):
+    obj = {}
+    for idx, (name, _type) in enumerate(zip(schema.names, schema.fields)):
+        if isinstance(data, dict):
```

Comment:
why not check for `Struct` and use the existing attributes set in `__dict__` , as opposed to creating a duplicate dictionary for the same data (struct.fields) ?